modules/removing-filters-from-insights-advisor-recommendations.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * support/remote_health_monitoring/using-insights-to-identify-issues-with-your-cluster.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="removing-filters-from-insights-recommendations_{context}"]
+= Removing filters from Insights Advisor recommendations
+
+You can apply multiple filters to the list of recommendations and remove them individually or completely reset them.
```

Comment:
```suggestion
You can apply multiple filters to the list of recommendations or remove them individually. You can also reset the filters.
```
- Edit for minimalism and to eliminate the awkward grammatical construction.

Reply:
done